### PR TITLE
LINE, Slack, Telegram, Viber Example Rewrite

### PR DESCRIPTION
**Pull request by chentsulin**
@Yoctol/sd Please help review this PR.

If everything looks good, I will apply those changes to all of examples.

### CODE REVIEW

File: examples/telegram-builder/README.md [hunk 1]
```diff
@@ -7,15 +7,41 @@ Download this example or clone [bottender](https://github.com/Yoctol/bottender).
...
+
+You must put `accessToken` into `bottender.config.js`.
+
+After that, you can run the bot with this npm script:
+
+```
 npm run dev
 ```
 
+This command will start a development bot server at `http://localhost:5000`.
```
Comment (wtflink) [inline examples/telegram-builder/README.md]: I feel like this line could be 
``` This command will start server for bot developing at `http://localhost:5000`.```

**Comment by chentsulin** [comment]
Ready for reviews. I will open a new PR for messenger.

File: examples/telegram-builder/README.md [hunk 2]
```diff
@@ -7,15 +7,41 @@ Download this example or clone [bottender](https://github.com/Yoctol/bottender).
...
+
+## Set webhook
+
+To set the webhook, run the following command with global `bottender`:
```
Comment (wtflink) [inline examples/telegram-builder/README.md]: ```While the server running, you can run following command with global `bottender` to set up  the webhook :```

File: examples/line-builder/README.md [hunk 3]
```diff
@@ -7,14 +7,34 @@ Download this example or clone [bottender](https://github.com/Yoctol/bottender).
 ```
 curl https://codeload.github.com/Yoctol/bottender/tar.gz/master | tar -xz --strip=2 bottender-master/examples/line-builder
 cd line-builder
+```
+
+Install it:
```
Comment (tw0517tw) [inline examples/line-builder/README.md]: should these be sth like `install dependencies`?

**Pull request merged**